traffic_ops/testing/api_contract/v4/conftest.py

```diff
@@ -125,9 +127,14 @@ def pytest_addoption(parser: pytest.Parser) -> None:
 		default=os.path.join(os.path.dirname(__file__), "to_data.json")
 	)
 	parser.addoption(
-		"--prerequisites",
-		help="Path to prerequisites file.",
-		default=os.path.join(os.path.dirname(__file__), "prerequisite_data.json")
+		"--request-template",
+		help="Path to request prerequisites file.",
+		default=os.path.join(os.path.dirname(__file__), "request_template.json")
+	)
+	parser.addoption(
+		"--response-template",
+		help="Path to response prerequisites file.",
+		default=os.path.join(os.path.dirname(__file__), "response_template.json")
 	)
```

Comment:
Why is this duplicated?

Reply:
It's not duplicated. One for request template and another for response template.